PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,14 +1,8 @@
-Ticket [JIRA-CODE](link) 
+Ticket [JIRA-CODE](link)
 
 ---
 
 **Description**
 
 
 **Changes in detail, screenshots, risks, anything else to add?**
```

Comment:
Won't we still need anything anymore, not even labels?

Reply:
The only thing we will need going forward is just *one* label! 🤩 

All five repos will now have a script that checks that the PR has one of the 'versioning' labels applied (see `pr-label.yml` in [this PR](https://github.com/FairHQ/dashboard/pull/599/files) for example). This is part of the checks of the PR, so the PR will be red if none or more than one of the versioning labels are applied.

To me this feels like enough of a protection, and that the checkboxes in the PR template might just be duplication (and one more manual thing to remember to do). What do you think? 🤔

Reply:
Makes sense. The red PR will do 👍  Good job!